package.json

```diff
@@ -179,7 +179,7 @@
     "babel-plugin-dev-expression": "^0.2.1",
     "babel-plugin-react-docgen": "^2.0.0",
     "bowser": "^1.6.1",
-    "carbon-components": "^9.69.0",
+    "carbon-components": "^9.78.0",
```

Comment:
is this PR dependent on https://github.com/IBM/carbon-components/pull/1933? both netlify and local build don't appear to reposition the label when the knob is modified

Reply:
@emyarod Using the knob in experimental requires 
[IBM/carbon-components#1933](https://github.com/IBM/carbon-components/pull/1933) and that's why it doesn't work with Netlify link. That said, to test it locally, it needs to be with non-experimental mode and pull in `9.78.0`, which contains IBM/carbon-components#1722. Please don't hesitate to ping me if you still see the problem - Thanks!